.github/workflows/release-creator.yml

```diff
@@ -0,0 +1,28 @@
+name: Release Creator
```

Comment:
This workflow is only needed in this repo, this one will loop over specified repo's and create branches.